doc/nrf/gs_programming.rst

```diff
@@ -102,7 +102,7 @@ Complete the following steps to build |NCS| projects with SES after :ref:`instal
    .. important::
       If you are working with an nRF9160 DK, make sure to select the correct controller before you program the application to your development kit.
 
-      Put the **SW5** switch (marked debug/prog) in the **NRF91** position to program the main controller, or in the **NRF52** position to program the board controller.
+      Set the **SW5** switch (marked debug/prog) in the **NRF91** position to program the main controller, or in the **NRF52** position to program the board controller. In newer versions of the DK, the switch is **SW10**.
```

Comment:
We should turn this around - we want the current version as default, and the older version as "btw, it might also be called SW5".
And add the version information - "in DK vX.X and earlier". Not sure when this was changed, you could ask around or check the HW user guide.

Also, new line for the new sentence.

Reply:
The weirdest thing was that this failed the gh compliance check when I added the new sentence on a new line. Changed it to the same line and it passed the check.

Reply:
Probably you left the space at the end of the first line?

Reply:
Nope

Reply:
I will push the suggested changes as soon as someone tells me the version from which the switch changed. I did not find that information in the infocenter.

Reply:
Got the version info from Rune Brandsegg.